helm/README.md

```diff
@@ -0,0 +1,48 @@
+# Ballista Helm Chart
+
+Helm is a convenient way to install applications into Kubernetes. It can work against any Kubeneretes environement,
+including all the major cloud providers. 
+For the purposes of this documentation, we will use Kubernetes-in-Docker (kind) to install locally.
+
+## Prerequisites
+
+```shell
+sudo apt install kubectl docker.io docker-compose
+sudo snap install go --classic
+go install sigs.k8s.io/kind@v0.16.0
+~/go/bin/kind create cluster
+```
+
+## Build
+
+```shell
+dev/build-ballista-docker.sh
```

Comment:
This fails for me with:

```
ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
```

Maybe we can downgrade to 3.3 unless there are specific features we need from newer versions?

Reply:
I suppose there is a wider discussion we should have about what minimum versions of different operations systems we plan on supporting.

Reply:
https://github.com/apache/arrow-ballista/issues/328

Reply:
PR to downgrade to version 3.3 - https://github.com/apache/arrow-ballista/pull/329